platform/src/main/resources/config/anchor-asset-default-values.yaml

```diff
@@ -0,0 +1,155 @@
+#
+# This file contains the configuration for the assets. The values used here are for testing purposes only.
+#
+
+# The list of the assets that the anchor supports.
+#
+# Note: The `id` of each asset is the unique identifier for the asset which is formated as `schema:code:issuer`.
+#    - `schema`: is the schema of the asset. It can be either `stellar` or `iso4217`.
+#    - `code`: is the code of the asset.
+#    - `issuer`: is the issuer of the asset. It is only used for Stellar assets that are not native.
+# Currently, the anchor supports the following asset schemas:
+#   - `stellar`: The Stellar asset.
+#   - `stellar:native`: The native asset (XLM).
+#   - `iso4217`: The fiat asset.
+#
+assets:
+  # The Stellar asset that is not native (XLM).
+  - id: "stellar:"
+    # The `distribution_account` field is the Stellar account that the anchor will use to distribute the asset.
+    distribution_account:
+    # The `significant_decimals` field is the number of decimals that the asset supports. It must not be defined if the asset is XLM (stellar:native).
+    # default: 7
+    significant_decimals: 7
```

Comment:
I see we are removing `sep38.decimals` and only use `significant_decimals`?
Maybe some comment about why's the default value

Reply:
Typo.  Removed.
```
  #  default:7
```